alerts/openshift-virtualization-operator/WaspNamespaceDetected.md

```diff
@@ -0,0 +1,45 @@
+# WaspNamespaceDetected
+
+## Meaning
+
+This alert fires when there is a duplicate wasp-agent deployment on the cluster
+
+## Impact
+Inconsistent behavior of workload swap configuration.
```

Comment:
Please give examples if you can.
Like:
For example, in case memory overcommit was set in the duplicate wasp-agent and in the new one there is a different value it can lead to...

Reply:
Fixed